lib/pbench/server/api/__init__.py

```diff
@@ -70,6 +72,50 @@ def create_app():
     return app
 
 
+def _build_cors_preflight_response():
+    response = make_response()
+    response.headers.add("Access-Control-Allow-Origin", "*")
+    response.headers.add("Access-Control-Allow-Headers", "*")
+    response.headers.add("Access-Control-Allow-Methods", "*")
+    return response
+
+
+def _corsify_actual_response(response):
+    response.headers.add("Access-Control-Allow-Origin", "*")
+    return response
+
+
+class Download(Resource):
+    def post(self):
+        global app
+        try:
+            # query ElasticSearch
+            json_data = request.get_json()
+            if "payload" in json_data:
+                response = requests.post(json_data["url"], json=json_data["payload"])
+            else:
+                response = requests.get(json_data["url"])
```

Comment:
Can we make it so that the URL does not include the host name of the Elasticsearch instance, but instead have the server provide that, but the client just provides the path and arguments components?